src/main/kotlin/org/rust/ide/inspections/import/AutoImportFix.kt

```diff
@@ -52,10 +53,10 @@ class AutoImportFix(element: RsElement) : LocalQuickFixOnPsiElement(element), Hi
         }
 
         if (candidates.size == 1) {
-            importItem(project, candidates.first(), element)
+            element.containingMod.importItem(candidates.first())
```

Comment:
Hm, here we assume that `AutoImportFix` starts in write action and it's true at this moment and that's why this call is not wrapped into write action
But it's wrong and we should avoid running the whole action in write action because it can do a lot of things in `findApplicableContext`
So let's add `override fun startInWriteAction(): Boolean = false` and wrap this call into write action

Reply:
Some tests will fail :(

Reply:
Ok, I'll take a look after this PR